client/my-sites/upgrades/domain-management/name-servers/dns-templates.jsx

```diff
@@ -2,23 +2,53 @@
  * External dependencies
  */
 import React, { Component } from 'react';
-import { find } from 'lodash';
+import { find, replace } from 'lodash';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import ButtonGroup from 'components/button-group';
 import Card from 'components/card';
+import { dnsTemplates } from 'lib/domains/constants';
 import DnsTemplateButton from './dns-template-button';
-import Office365 from '../dns/office-365';
+import EmailProvider from '../dns/email-provider';
 
 class DnsTemplates extends Component {
 	constructor( props ) {
 		super( props );
+		const { translate } = this.props;
 		this.state = {
 			currentComponentName: null,
+			currentProviderCardName: null,
 			templates: [
-				{ name: 'Office 365', Component: Office365 }
+				{
+					name: 'G Suite',
+					label: translate( 'G Suite Verification Token - from the TXT record verification' ),
```

Comment:
Technically, the `Office 365` part could become a parameter for the translation and then the translation could be shared between the two cases (Office 365 and G Suite). `Office 365` and `G Suite` are product names and should not be translated either way (at least, I'm sure about that for `G Suite`, probably true for O365 too).
Not sure if it's actually a good practice, though - @yoavf, my i18n guru, what's your opinion? 🙇